.github/workflows/build.yaml

```diff
@@ -66,3 +66,45 @@ jobs:
       node_type: "gpu-v100-latest-1"
       run_script: "ci/build_docs.sh"
       sha: ${{ inputs.sha }}
+  wheel-build-cpp:
```

Comment:
Looks like the `build.yaml` uses `wheel-build-<type>`, but `pr.yaml` is `wheel-<type>-build`. 

Looks like https://github.com/rapidsai/rmm/pull/1554 went with `wheel-build-<type>`, so maybe use that in `pr.yaml`.

Reply:
Actually, it looks like the conda jobs are `<type>-build` or `conda-<type>-build`, so I'm not sure what to say.

Reply:
I've added aligning these names as a task to https://github.com/rapidsai/build-planning/issues/44. I think we can merge this in whatever form we decide, then do a universal cleanup as part of that issue.